### PR TITLE
Add vivotek.markdown

### PR DESCRIPTION
**Pull request by HarlemSquirrel**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26071

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10189"></a>

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
You could put an advanced config at the bottom with optional variables.

File: source/_components/vivotek.markdown [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+title: "Vivotek Camera"
+description: "Instructions on how to integrate Vivotek cameras within Home Assistant."
+ha_category:
+  - Camera
+logo: vivotek.jpg
+ha_release: 0.97
```
Comment (MartinHjelmare) [inline source/_components/vivotek.markdown]: 0.99

File: source/_components/vivotek.markdown [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  type: boolean
+verify_ssl:
+  description: Enable or disable SSL certificate verification. Set to false to use an http-only camera, or you have a self-signed SSL certificate and haven't installed the CA certificate to enable verification.
+  required: false
+  default: true
+  type: boolean
+framerate:
+  description: The number of frames-per-second (FPS) of the stream. Can cause heavy traffic on the network and/or heavy load on the camera.
+  required: false
+  type: integer
```
Comment (MartinHjelmare) [inline source/_components/vivotek.markdown:60]: Add the default for the framerate (2).

File: source/_components/vivotek.markdown [hunk 3]
```diff
@@ -0,0 +1,140 @@
+---
+title: "Vivotek Camera"
+description: "Instructions on how to integrate Vivotek cameras within Home Assistant."
+ha_category:
+  - Camera
+logo: vivotek.jpg
+ha_release: 0.97
+ha_iot_class: Configurable
```
Comment (MartinHjelmare) [inline source/_components/vivotek.markdown]: I'd say it's `Local Polling`.

**Review by klaasnicolaas — DISMISSED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @HarlemSquirrel! Looking great now! 👍

**Pull request merged**